db/1.9/modules/ROOT/pages/database-documentation.adoc

```diff
@@ -0,0 +1,1161 @@
+| Target Variable a| String |  The name of a variable to store the operation's output. |  |
+| Target Value a| String |  An expression to evaluate against the operation's output and store the expression outcome in the target variable |  `#[payload]` |
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors. |  |
+|===
+
+==== Output
+[%autowidth.spread]
+|===
+|Type |Array of Object
+|===
+
+=== For Configurations
+* <<config>>
+
+=== Working with Pooling Profiles
+When working with pooling profiles and the *Select* operation, the connection remains open until the flow execution ends or the content of the streams are consumed completely, or if the connection is the transaction key. This behavior occurs because the resultset the operation generates can have a stream or be part of an ongoing transaction.
+
+
+==== Throws
+
+* DB:BAD_SQL_SYNTAX
+* DB:CONNECTIVITY
+* DB:QUERY_EXECUTION
+
+[[querySingle]]
+=== Query Single
+`<db:query-single>`
+
+
+This operation selects a single data record from a database. If you provide an SQL query that returns more than one row, then only the first record is processed and returned. This operation does not use streaming, which means that immediately after performing the single query operation, the complete content of the selected record is loaded into memory.
+
+==== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use | | x
+| Transactional Action a| Enumeration, one of:
+
+** ALWAYS_JOIN
+** JOIN_IF_POSSIBLE
+** NOT_SUPPORTED |  The type of joining action that operations can take regarding transactions |  `JOIN_IF_POSSIBLE` |
+| Streaming Strategy a| * <<repeatable-in-memory-iterable>>
+* <<repeatable-file-store-iterable>>
+* non-repeatable-iterable |  Configure to use repeatable streams |  |
+| Query Timeout a| Number |  Indicates the minimum amount of time before the JDBC driver attempts to cancel a running statement. By default, no timeout is used. |  `0` |
+| Query Timeout Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  A time unit that qualifies the `#queryTimeout` |  `SECONDS` |
+| Fetch Size a| Number |  Indicates how many rows to fetch from the database when rows are read from a ResultSet. This property is required when streaming is set to `true`, in which case a default value of `10` is used. |  |
+| Max Rows a| Number |  Sets the limit for the maximum number of rows that any ResultSet object generated by this message processor can contain for the given number. If the limit is exceeded, the excess rows are silently dropped. |  |
```

Comment:
I suggest:  "The maximum number of rows that any ResultSet object generated by this message processor can contain."